test/attr.test.js

```diff
@@ -55,11 +55,11 @@ describe('#attr', () => {
     it('fails when the actual does not match the expected', (wrapper) => {
       expect(() => {
         expect(wrapper).to.have.attr('id', 'invalid')
-      }).to.throw(`to have a 'id' attr`)
+      }).to.throw("to have a 'id' attr")
```

Comment:
it's kind of strange that the linter would require double quotes over backticks, when you use single-quoted strings elsewhere. why is this?